packages/vechain/src/ConnexProviderWrapper.ts

```diff
@@ -40,8 +46,18 @@ export class ConnexProviderWrapper extends EventEmitter implements EthereumProvi
             });
     }
 
+    public async getVechainNetwork(): Promise<Network> {
+        const provider = await this._provider;
+        return {
+            name: this._networkName,
+            chainId: this._networkConfig.chainId!!,
+            _defaultProvider: (providers: any, options?: any) => provider
+        }
+    }
+
     public async sign(transaction: Deferrable<TransactionRequest>) {
         let key: Wallet.Key | undefined = undefined;
+        console.log(`This is the tx ${JSON.stringify(transaction)}`);
```

Comment:
As far as I see above we have a logger for debug, should we have another one for errors instead of `console.log`?

Reply:
This was just for debugging purposes, removed